consensus/polybft/contractsapi/helper_test.go

```diff
@@ -22,11 +22,6 @@ func TestEncoding_SignedTypes(t *testing.T) {
 			Signature:       [2]*big.Int{big.NewInt(1), big.NewInt(2)},
 			Bitmap:          big.NewInt(1).Bytes(),
 		},
-		&SignedBridgeMessageBatch{
```

Comment:
Add this back to the test. And remove the `signedType` interface and just use `ABIEncoder`.